code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -204,68 +202,67 @@
 	var/obj/item/sample_object
 	var/number
 
-	New(obj/item/sample as obj)
+	New(obj/item/sample)
```

Comment:
Gross way to declare procs.
Best to write this as `/datum/numbered_display/New(...)`